src/Components/TestComponent.js

```diff
@@ -0,0 +1,11 @@
+const TestComponent = ({shops}) => {
+    return(
+        <div className="list">
+            {Object.values(shops).map((item, index) => (
```

Comment:
you work with array and don't need to use here Object.values. To have correct HTML document you need to use ```<ul><li></li></ul>``` to render list